clients/tests/simple_test.cpp

```diff
@@ -645,3 +708,29 @@ TEST(hipfftTest, OutplaceOnly)
     fftw_destroy_plan(ref_p);
     fftw_free(ref_out);
 }
+
+TEST(hipfftTest, GetVersion)
+{
+    // valid use case(s)
+    int tmp;
```

Comment:
Would it be worth initializing `tmp` to 0 and checking that it got set to a nonzero value after?